content/en/_index.md

```diff
@@ -730,6 +661,80 @@ $dlp = new DlpServiceClient();
 {{< /tab >}}
 {{< /tabpane >}}
 
+#### Authentication {#authentication}
+
+Samples should always authenticate using [Application Default Credentials][ADC].
+Most clients do this automatically, and no special steps are required.
+
+If, for whatever reason, the code snippet requires specific credentials,
+then, apart from showing how to obtain such credentials, it should show
+hot to intialized a client with other than ADC.
+
+[ADC]:https://cloud.google.com/docs/authentication/production#automatically
```

Comment:
Update link to:
https://cloud.google.com/docs/authentication/external/about-adc

Reply:
Done